lldb/source/Plugins/SymbolFile/DWARF/DWARFUnit.cpp

```diff
@@ -655,7 +655,7 @@ DWARFUnit::GetDIE(dw_offset_t die_offset) {
 
   if (!ContainsDIEOffset(die_offset)) {
     GetSymbolFileDWARF().GetObjectFile()->GetModule()->ReportError(
-        "GetDIE for DIE {0:x16} is outside of its CU {0:x16}", die_offset,
+        "GetDIE for DIE {0:x16} is outside of its CU {1:x16}", die_offset,
```

Comment:
I would push the fixes to main directly ahead of the PR, so that any revert of this PR does not lose these fixes :)

Reply:
Yeah, that was one of my plans, so split out independent fixes as a separate patch before.

Reply:
Resolving.